applications/ConvectionDiffusionApplication/python_scripts/convection_diffusion_base_solver.py

```diff
@@ -98,6 +98,18 @@ def __init__(self, model, custom_settings):
             self.main_model_part.ProcessInfo.SetValue(KratosMultiphysics.DOMAIN_SIZE, domain_size)
             self.solver_imports_model_part = True
 
+        element_name = self.settings["element_replace_settings"]["element_name"].GetString()
+        if (self.settings["use_orthogonal_subscales"].GetBool() is True):
```

Comment:
This is something specific of the explicit element. I think that at this point it must be moved to the explicit new solver.

Reply:
Ok. I will add the flag "use_orthogonal_subscales" to the explicit solver. That should not be difficult to move in the base_solver when there will be other OSS-compatible elements.